### PR TITLE
Security #1: Attempt to bump up `generic-array`

### PR DESCRIPTION
**Pull request by michalfita**
Addresses report from dependabot: https://github.com/atsams-rs/atsamx7x-rust/security/dependabot/1

### CODE REVIEW

**Comment by tmplt** [comment]
The linked report does not exist: 404.

**Comment by michalfita** [comment]
image
That's pretty interesting, so for some bizarre reason you don't have access rights to it.

**Comment by michalfita** [comment]
What about now?

File: hal/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "atsamx7x-hal"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (tmplt) [inline hal/Cargo.toml]: Techically, a MINOR bump is overkill. A PATCH bump suffices.

File: hal/Cargo.toml [hunk 2]
```diff
@@ -21,9 +21,9 @@ readme = "README.md"
 edition = "2021"
 
 [dependencies]
-cortex-m = "0.7"
+cortex-m = "0.7.5"
```
Comment (tmplt) [inline hal/Cargo.toml:24]: Commit does not mention a `cortex-m` bump.

**Review by tmplt — CHANGES_REQUESTED** [review]
Still can't access the report. But both dep bumps LGTM. I'd rewrite the commit:

```
cargo: bump nb, cortex-m

nb bumped due to <link to report>.
```

or similar.

**Review by tmplt — APPROVED** [review]

**Pull request merged**